jwst/cube_build/src/cube_match_sky_pointcloud.c

```diff
@@ -513,7 +513,7 @@ static PyObject *cube_wrapper(PyObject *module, PyObject *args) {
       (!(coord2 = ensure_array(coord2o, &free_coord2))) ||
       (!(wave = ensure_array(waveo, &free_wave))) ||
       (!(flux = ensure_array(fluxo, &free_flux))) ||
-      (!(err = ensure_array(erro, &free_err))) ||
+      (!(err = ensure_array(err_o, &free_err))) ||
```

Comment:
Similar to the other comments `erro` fits with `fluxo`, `waveo`, etc. I vote for updating all or none.